erpnext/stock/doctype/pick_list/pick_list.py

```diff
@@ -169,6 +169,13 @@ def update_package_tag(self, reset=False):
 			if not item.package_tag:
 				continue
 
+			if item.package_tag and item.coa_batch_no:
```

Comment:
check if Outgoint TransferPackage Tag is selected and COA Batch fetched ftom Source Package tag then only allow to update